plugins/action_revoke_permissions/plugin.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "action_revoke_permissions",
+  "code": "revokePermissions",
+  "name": "Revoke Permissions",
+  "description": "Revokes existing permission(s) from user/org -- only sets existing permissions to inactive, doesn't actually delete records",
+  "required_parameters": ["username", "permissionNames"],
+  "optional_parameters": ["orgName", "orgId"],
+  "output_properties": ["permissionJoinIds", "permissionNames"],
```

Comment:
It would be nice to output `permissionJoinIds` and `permissionNames`, but currently it's outputs other fields. Look what I'm getting after running the action (Postman):
```
{
    "status": "SUCCESS",
    "error_log": "",
    "output": {
        "revokedPermissions": [
            {
                "permission_name_id": 7,
                "name": "applyDrugRegoMMC"
            },
            {
                "permission_name_id": 8,
                "name": "applyDrugRegoMMD"
            }
        ]
    }
}
```

Reply:
The permissionName is there (just "name"), but yeah okay, I've updated to include the permissionJoinIds and renamed "name" to "permissionName"

091f12f9